code/modules/admin/db_ban/functions.dm

```diff
@@ -661,14 +658,18 @@
 					output += "<td align='center' colspan='5' bgcolor=''><b>UNBANNED by admin [unbanckey] on [unbantime][unban_round_id ? " (Round [unban_round_id])" : ""]</b></td>"
 					output += "</tr>"
 				output += "<tr>"
-				output += "<td colspan='5' bgcolor='white'>&nbsp</td>"
+				output += "<td colspan='5' bgcolor='#151515'>&nbsp</td>"
```

Comment:
Where is this being used, because the background on browsers is #272727

![image](https://user-images.githubusercontent.com/25063394/210658247-bfc7ae4e-feb1-4c6a-a513-ca7bc1bc0aad.png)

Reply:
That specific one you comment on is a space between things, there is no content there.